software/speller/data_collection_platform/README.md

```diff
@@ -25,6 +27,7 @@ $ flask run
 
 
 ## Connecting to the database
+TODO: Update this part
```

Comment:
does this section still need to be done before we merge?

Reply:
no I don't think so, its purpose is to communicate instructions for the data, signal, and ml teams to connect to the databse using psql, we can do this later.